package.json

```diff
@@ -51,7 +51,8 @@
   },
   "private": true,
   "dependencies": {
-    "espree": "5.0.1"
+    "espree": "5.0.1",
+    "kolibri-design-system": "https://github.com/nucleogenesis/kolibri-design-system-1#inline-svg"
```

Comment:
This will eventually point to the published kolibri-design-system package.

Reply:
can be updated now - I'll point it at an alpha:

https://github.com/learningequality/kolibri-design-system/releases/tag/v0.2.0-alpha0